AliveSFU/AliveSFU/ExerciseTileViewController.swift

```diff
@@ -26,7 +28,19 @@ class ExerciseTileViewController: UIViewController {
         self.removeAnimate()
         //self.view.removeFromSuperview()
     }
-
+    @IBAction func background1(_ sender: Any) {
+        self.removeAnimate()
+    }
+    @IBAction func background2(_ sender: Any) {
+        self.removeAnimate()
+    }
+    @IBAction func background3(_ sender: Any) {
+        self.removeAnimate()
+    }
+    @IBAction func background4(_ sender: Any) {
+        self.removeAnimate()
+    }
```

Comment:
You won't need these after implementing the view as a popover. Also remove them from the storyboard